src/sentry/static/sentry/app/api.jsx

```diff
@@ -191,13 +191,12 @@ export class Client {
     return this.activeRequests[id];
   }
 
-  requestPromise(path, options = {}) {
+  requestPromise(path, {includeAllArgs, ...options} = {}) {
     return new Promise((resolve, reject) => {
       this.request(path, {
         ...options,
         success: (data, ...args) => {
-          // This fails if we need jqXhr :(
-          resolve(data);
+span class="pl-smi">           includeAllArgs ? resolve([data, successString, allArgs]) : resolve(data);
```

Comment:
Shouldn't this be `resolve([data, ...args])` to match?

Also is `span class="pl-smi">` a typo?

Reply:
yeah it was.